chart/tests/git-sync-scheduler_test.yaml

```diff
@@ -1,154 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is it expected?

Reply:
Yes. I replaced all tests in this file with the python file. I plan to replace the rest of the tests in future PRs and then remove helm-unittest completely.

Reply:
Ok. Thanks.

I've already converted the rest of the files.
https://github.com/apache/airflow/pull/11827